priv_events/app/controllers/events_controller.rb

```diff
@@ -0,0 +1,39 @@
+class EventsController < ApplicationController
+    def index
+        @events = Event.all
+        @upcoming_events = @events.where('date >= ?', Date.today)
+        @prev_events = @events.where('date < ?', Date.today)
```

Comment:
- [ ] scope for upcoming events  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/scopes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
- [ ] scope for previous events  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/scopes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))